src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java

```diff
@@ -79,4 +84,39 @@ public CarinaDescriptionPageBase login() {
         return clickLoginBtn();
     }
 
+    @Override
+    public boolean isPageOpened() {
+        return nameInputField.isElementPresent() &&
+               passwordInputField.isElementPresent() &&
+               maleRadioBtn.isElementPresent() &&
+               femaleRadioBtn.isElementPresent() &&
+               privacyPolicyCheckbox.isElementPresent() &&
+               loginBtn.isElementPresent();
+    }
```

Comment:
Its bad practice, better when you verify only one specific element on this page
Example: title CARINA

Reply:
done